### DISCUSSION TITLE
Is possible to have the (+) withot password?

### DISCUSSION THREAD

**Discussion by pixartmx** [YOURLS/YOURLS#3530] [Q&A] [answered]
I need to share with my clients there own shortlink and there + option to see there stats in real time but is asking for user and password and i dont what to let them see the admin.

i try to add what someone suggest to unlocked the statistics (+) for the public

define('YOURLS_PRIVATE_INFOS', true); define('YOURLS_NOSTATS', false); 

but isnt working

how can solve this problem I'be grateful for any help!

**Comment by dgw** [comment, marked as answer]
> define('YOURLS_PRIVATE_INFOS', true);

This is causing the password requirement. Set it to false.

**Reply by pixartmx** [reply]
thank you works perfect
